include/fc/io/cfile.hpp

```diff
@@ -15,7 +15,7 @@
 namespace fc {
 
 namespace detail {
-   using unique_file = std::unique_ptr<FILE, decltype( &fclose )>;
+   using unique_file = std::unique_ptr<FILE>;
```

Comment:
This is definitely wrong.  You can't `delete` a `FILE*`, which is what the vanilla `unique_ptr` will try to do.

Reply:
right- it seems to me that the unique_ptr is only complicating things.